hivemq-edge/src/frontend/src/extensions/datahub/components/helpers/PolicyErrorReport.tsx

```diff
@@ -0,0 +1,64 @@
+import { FC } from 'react'
+import {
+  Accordion,
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Box,
+  Button,
+  ButtonGroup,
+  Text,
+  VStack,
+} from '@chakra-ui/react'
+import { ProblemDetailsExtended } from '@/api/types/http-problem-details.ts'
+import { useTranslation } from 'react-i18next'
+
+interface PolicyErrorReportProps {
+  errors: ProblemDetailsExtended[]
+  onFitView?: (id: string) => void
+  openConfig?: (id: string) => void
+}
+
+const PolicyErrorReport: FC<PolicyErrorReportProps> = ({ errors, onFitView, openConfig }) => {
+  const { t } = useTranslation('datahub')
+  const status = errors.length ? 'warning' : 'success'
```

Comment:
question(non-blocking): Is status used across the module? if yes, then I would suggest to use an `as const` or `enum` type.

Reason is prevention of magic strings

```suggestion
  const status = errors.length ? 'warning' : 'success'
```

Reply:
redesigned later on